style.css

```diff
@@ -0,0 +1,839 @@
+  }
+
+  .navbar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #fff;
+    box-shadow: 5px 10px 8px #d4d4d4;
+  }
+
+  .navbar img {
+    margin-left: -4rem;
+    margin-top: -0.8rem;
+  }
+
+  .navbar .nav-items {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-direction: row;
+    list-style: none;
+    padding: 0;
+    height: 0;
+    width: 0;
+    gap: 3rem;
+    background-color: #fff;
+    opacity: 1;
+    transform: none;
+    margin-top: 0.9rem;
+  }
+
+  .navbar ul a {
+    text-decoration: none;
+    color: #5c5a5a;
+    margin: 0;
+    font-size: 16px;
+  }
+
+  .see-the-whole-program p a {
+    text-decoration: underline;
+    color: #e9e9ea;
+    text-transform: uppercase;
+  }
+
+  .navbar ul li a:hover {
+    color: #ec5242;
+    font-size: 18px;
+    animation: 0.5s ease;
+  }
+
+  .navbar .nav-button {
+    border-color: #ec5242;
+    border-width: 7px;
+    color: #ec5242;
+    background-color: #fff;
+    font-size: 16px;
+    padding: 12px;
+    margin: -0.5rem 15rem 0 1rem;
+  }
+
+  .navbar .nav-button:hover {
+    color: #fff;
+    background-color: #ec5242;
+    font-size: 18px;
+  }
+
+  .header-content-container {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    background-image: url(img/desktop-bg.png);
+    background-size: cover;
+    padding: 35rem 0 0;
+    margin: 0 0 0 -1rem;
+    height: 100%;
+  }
+
+  .header-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    flex-wrap: wrap;
+    align-items: flex-start;
+    margin: -53rem 0 0 19rem;
+  }
+
+  .header-content .content-title {
+    display: flex;
+    justify-content: flex-start;
+    color: #ec5242;
+    font-size: 40px;
+    font-weight: 400;
+    text-align: center;
+    margin-top: -1rem;
+    margin-left: -23rem;
+  }
+
+  .header-content h1 {
+    font-family: "COCOGOOSE", sans-serif;
+    background-image: url(img/red.png);
+    background-size: cover;
+    background-position: bottom;
+    color: transparent;
+    font-weight: 700;
+    font-size: 60px;
+    background-clip: border-box;
+    -webkit-background-clip: text;
+    margin-left: 3rem;
+    align-items: start;
+  }
+
+  .header-content .header-content-body {
+    background-color: #f7f7f7;
+    border: 0.28rem solid #fff;
+    display: flex;
+    text-align: center;
+    align-items: center;
+    height: auto;
+    padding: 5px;
+    margin-left: 7rem;
+    width: 80%;
+  }
+
+  .header-content-body p {
+    display: flex;
+    font-size: 15px;
+    line-height: 2rem;
+    padding-top: 0.6rem;
+  }
+
+  .header-content .summit-date {
+    color: #555;
+    font-weight: 800;
+    font-size: 30px;
+    padding-top: 1.5rem;
+    padding-left: 6rem;
+  }
+
+  .header-content .summit-venue {
+    color: #555;
+    font-weight: 150;
+    font-size: 20px;
+    padding-top: 0.3rem;
```

Comment:
- [ ] As suggested by the previous reviewers, here your elements in the hero section are not responsive and also they are not in the right position as in the original design. 
I think the main reason is that you are repeating a lot of CSS for each and every little element that the css styling of those elements are overlapping with each other and going against responsive design. kindly consider making your design responsive and try to adjust your elements so that they look similar to the original design.
your design
<img width="923" alt="Screenshot 2023-07-08 at 10 36 09 PM" src="https://github.com/Edgahkipkemoi/capstone/assets/96683843/2727d3e5-e661-4d5d-9d06-c25e852f278a">